Common/src/Storage/ITaskTable.cs

```diff
@@ -251,6 +251,20 @@ public interface ITaskTable : IInitializable
                                          Expression<Func<TaskData, T>>    selector,
                                          CancellationToken                cancellationToken = default);
 
+  // TODO Should be compatible with EFCORE : https://learn.microsoft.com/en-us/ef/core/saving/execute-insert-update-delete#updating-multiple-properties
```

Comment:
If this to do in this PR or in another ?

Reply:
It should be working with EF Core but the adapter for ef core will be in another pr